src/components/fold-media.css

```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```

Comment:
I started with what @d4tocchini has here and scaled way back on the overlap: 
![image](https://cloud.githubusercontent.com/assets/440221/14207899/bd373e4e-f7ea-11e5-999d-4a908f8da15c.png)